test/include/seqan3/test/seqan2.hpp

```diff
@@ -35,11 +35,7 @@
 #include <seqan/basic.h>
 #include <seqan3/std/ranges>
```

Comment:
```suggestion
#include <range/v3/range_fwd.hpp>
#include <seqan/basic.h>
#include <seqan3/std/ranges>
```

Not sure about the ordering here (I think you are the master in ordering includes xD)